development/file_formats/tscn.rst

```diff
@@ -63,7 +63,7 @@ so on. For example, a spatial node looks like:
 The Scene Tree
 --------------
 
-The scene tree is made up of ... nodes! The heading of each node consists of
+The scene tree is made up of … nodes! The heading of each node consists of
```

Comment:
IMO this conversion should be handled by the theme itself, not in the raw text files. But it's up for discussion.